setup.py

```diff
@@ -345,8 +345,8 @@ def get_exts_for(name):
         "attrs>=20; python_version>'2.7'",
         "attrs>=20,<22; python_version=='2.7'",
         "contextlib2<1.0; python_version=='2.7'",
-        "cattrs==1.0.0; python_version<'3.7'",
-        "cattrs~=1.1; python_version>='3.7'",
+        "cattrs<1.1.0; python_version<'3.7'",
+        "cattrs>=1.1.0; python_version>='3.7'",
```

Comment:
I don't think we need to restrict the cattrs version for `py>=3.7`. It should just default to a supported version.

```suggestion
        "cattrs; python_version>='3.7'",
```